src/olympia/migrations/938-drop-l10n-from-bio-user-profile.sql

```diff
@@ -0,0 +1,23 @@
+/* FOREIGN KEY is prod's - you will need to change this for your local DB */
+
+/* Add the new column and drop the constraint so we can clear the translations.*/
+ALTER TABLE `users`
+    ADD COLUMN `_bio` longtext NULL,
+    DROP FOREIGN KEY `users_ibfk_1`;
+
+/* Copy across the translated values.  There are some users that have multiple
+localizations so we're going to use the default locale (lang) localization. */
+UPDATE `users`
+    SET `_bio` = (
+        SELECT `localized_string` FROM `translations`
+        WHERE `id`=`bio` AND `locale`=`lang`);
+
+/* Clear out the now obsolete translations. */
+DELETE FROM `translations` WHERE `id` in (SELECT `bio` FROM `users`);
+
+/* Drop the old columns and rename bio_ to take the place of bio. */
+ALTER TABLE `users`
+    DROP COLUMN `bio`,
+    DROP COLUMN lang,
+    DROP COLUMN region,
+    CHANGE COLUMN `_bio` `bio` longtext NULL;
```

Comment:
For a 2 step migration I'm not clear how pt-online-schema-change will help us particularly  - the remaining issue would not be the time spent on the `rename` as such but the time from when the migration finishes to the code deployment finishes.
It would be:
1. add the new `_bio` column; copy across text; (change column in model to be `TextField` and refer to `_bio`)
2. drop `bio` column; rename `_bio` to `bio` (change column in model to `bio`)


Alternatively we could do a 3 step migration of:
1. add the new `_bio` column; copy across text; (change column in model to be `TextField` and refer to `_bio`)
2. drop old `bio` column; __copy__ all values from `_bio` to new `bio` column (change column in model to `bio`)
3. drop `_bio` 

This would prevent any downtime at the expense of hammering the DB by copying everything.  I'm not sure how pt-online-schema-change would help here either though.